parlai/agents/starspace/starspace.py

```diff
@@ -243,7 +238,7 @@ def observe(self, observation):
         return obs
 
     def same(self, y1, y2):
-        if len(y1.squeeze()) != len(y2.squeeze()):
+        if len(y1.squeeze(0)) != len(y2.squeeze(0)):
             return False
         if abs((y1.squeeze()-y2.squeeze()).sum().data.sum()) > 0.00001:
```

Comment:
the squeeze here maybe breaks

Reply:
I'll change it to `squeeze(0)` to just be safe